cmd/statusd/main.go

```diff
@@ -1,174 +1,142 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log"
 	"os"
-	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/status-im/status-go/geth/api"
 	"github.com/status-im/status-go/geth/params"
-	"gopkg.in/urfave/cli.v1"
 )
 
 var (
-	gitCommit  = "rely on linker: -ldflags -X main.GitCommit"
-	buildStamp = "rely on linker: -ldflags -X main.buildStamp"
-	app        = makeApp(gitCommit)
-	statusAPI  = api.NewStatusAPI()
+	gitCommit  = "N/A" // rely on linker: -ldflags -X main.GitCommit"
+	buildStamp = "N/A" // rely on linker: -ldflags -X main.buildStamp"
 )
 
 var (
-	// ProdModeFlag is whether we need dev or production settings
-	ProdModeFlag = cli.BoolFlag{
-		Name:  "production",
-		Usage: "Whether production settings should be loaded",
-	}
-
-	// NodeKeyFileFlag is a node key file to be used as node's private key
-	NodeKeyFileFlag = cli.StringFlag{
-		Name:  "nodekey",
-		Usage: "P2P node key file (private key)",
-	}
+	prodMode       = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile    = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir        = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID      = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
```

Comment:
We dont want to use `statuschain` anymore has default?

Reply:
It's default only for tests I guess.

Reply:
@influx6 that default value hasn't changed in this PR.

Reply:
Yes, it's only for tests. I was actually surprised to see it as a cli option, not sure if we want the caller to know about this chain. Anyway, I don't see any harm in it and it also make the options list more complete.